filters/block/block_test.go

```diff
@@ -69,7 +81,145 @@ func TestBlock(t *testing.T) {
 
 		})
 	}
+}
+
+func TestMatcherErrorCases(t *testing.T) {
+	t.Run("maxBufferAbort", func(t *testing.T) {
+		r := &nonBlockingReader{initialContent: []byte("fppppppppp .class")}
+		toblockList := []toblockKeys{{str: []byte(".class")}}
```

Comment:
If toblockList is the same and not relevant for the test then maybe make it a reusable var in the TestMatcherErrorCases

Reply:
fixed